csrc/scheduler/vectorize_helper.cpp

```diff
@@ -996,16 +996,16 @@ int64_t getVectorizationBreakPointOfReductionProducer(
   // Grab all the corresponding producer IDs that are mapped with the
   // innermost consumer IDs
   std::unordered_set<IterDomain*> producer_innermost_ids;
-  for (auto it = reduction_consumer->getMaybeRootDomain().begin() +
-           ((int64_t)reduction_consumer->nDims() - consumer_innermost_ndims);
```

Comment:
With DID loop split, reduction_consumer->nDims() might not equal reduction_consumer->getMaybeRootDomain().size().

Reply:
Is this the only change required to make it work with DID split? That's great, but a bit surprised...

Reply:
No -- I'm just checking in small changes along the way.